specification/reservations/resource-manager/readme.az.md

```diff
@@ -0,0 +1,12 @@
+# AZ reservations
+
+These settings apply only when `--az` is specified on the command line.
+
+``` yaml $(az)
+az:
+    extensions: quota
+    namespace: azure.mgmt.reservations
+    package-name: azure-mgmt-reservations
+az-output-folder: $(azure-cli-extension-folder)/src/reservations
+python-sdk-output-folder: "$(az-output-folder)/azext_reservations/vendored_sdks/quota"
+```
```

Comment:
based on the scenario that quota and reservation is functionally independent, I would suggest you to use multi-package https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/code-gen/configure-cli.md#multi-packages